lib/rpm_database.rb

```diff
@@ -132,11 +133,15 @@ def parse_stat_line(line)
       "dir"
     when "symbolic link"
       "link"
-    when "regular file"
+    when / file$/
```

Comment:
Instead of regular expression matching I would just cover the two cases we know about explicitly. But maybe that's too strict. When going for a fuzzy match it might make more sense to only match for `file$`, not for the space as well.